detections/endpoint/windows_expand_cabinet_file_extraction.yml

```
@@ -0,0 +1,85 @@
+name: Windows Expand Cabinet File Extraction
```

I also suggest the following name for clarity.

```suggestion
name: Windows Cabinet File Extraction Via Expand
```